src/App/index.js

```diff
@@ -11,7 +11,7 @@ const store = configureStore()
 const App = (props) => (
   <Provider store={store}>
     <Router>
-      <div className='pa4 mw8 center'>
```

Comment:
Moved to new `Main` component. I imagine most route views going forward will look like this from a high level:
```
<Nav ... />
<Main>
  <Split>
    ...
  </Split>
</Main>
```